frontend/packages/ceph-storage-plugin/src/components/create-backingstore-page/create-bs-page.tsx

```diff
@@ -5,13 +5,18 @@ import { Alert, AlertActionCloseButton, Title } from '@patternfly/react-core';
 import { history } from '@console/internal/components/utils/router';
 import { BreadCrumbs, resourcePathFromModel } from '@console/internal/components/utils';
 import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { useFlag } from '@console/shared/src/hooks/flag';
 import CreateBackingStoreForm from './create-bs';
 import '../noobaa-provider-endpoints/noobaa-provider-endpoints.scss';
+import { NooBaaBackingStoreModel } from '../../models';
+import { ODF_MODEL_FLAG } from '../../constants';
 
 const CreateBackingStoreFormPage: React.FC<CreateBackingStoreFormPageProps> = ({ match }) => {
   const { t } = useTranslation();
   const [showHelp, setShowHelp] = React.useState(true);
   const { ns, appName } = match.params;
+  const ODFInstalled = useFlag(ODF_MODEL_FLAG);
```

Comment:
```suggestion
  const isODF = useFlag(ODF_MODEL_FLAG);
```
Please update for all instances.